src/models/FGAtmosphere.h

```diff
@@ -232,9 +232,9 @@ class FGAtmosphere : public FGModel {
   /// Calculate the atmosphere for the given altitude.
   void Calculate(double altitude);
 
-  virtual double CalculateDensityAltitude(const double altitude) { return altitude; }
+  virtual double CalculateDensityAltitude(double density, double geoalt) { return geoalt; }
```

Comment:
I don't find the name `geoalt` very clear because both geometric and geopotential start with "geo".
And you dropped the `const`-ness of the parameters ?